### PR TITLE
enabling individual files DB Backup

### PR DESCRIPTION
**Pull request by zoide**
you may now choose whether backups should be one file or per DB.

### CODE REVIEW

File: templates/mysqlbackup.sh.erb [hunk 1]
```diff
@@ -16,9 +16,18 @@ DIR=<%= @backupdir %>
...
-find $DIR -mtime +${DAYS} -exec rm -f {} \;
-mysqldump -u${USER} -p${PASS} --opt --flush-logs --single-transaction \
- --all-databases <% if @backupcompress %>| bzcat -zc <% end %>> ${DIR}/mysql_backup_`date +%Y%m%d-%H%M%S`.sql<% if @backupcompress %>.bz2<% end  %>
+find $DIR -type f -mtime +${DAYS} -delete
 
+<%- if @onefile -%>
+FNAME="${DIR}/mysql_backup_${DATE}.sql<% if @backupcompress %>.bz2<% end  %>"
+${MYSQLDUMP} --all-databases <% if @backupcompress %>| bzcat -zc <% end %>> ${FNAME}
+<%- else -%>
+for DB in $( echo "show databases" |mysql -N ${COPTS} |grep -v '^#'); do
```
Comment (sorrison) [inline templates/mysqlbackup.sh.erb]: You'll need to change the mysql command to act like the mysqldump command here I think to pass in the credentials. Also what is ${COPTS}

**Comment by zoide** [comment]
Hi, fixed the credentials passing. I overlooked that, since my test machine ist passwordless :(

**Comment by sorrison** [comment]
Looks like you haven't pushed the changes up?

**Comment by zoide** [comment]
Yes I did. I squashed the commits into one. All seems fine to me.

https://github.com/NeCTAR-RC/puppet-mariadb/pull/24/commits/21be42b2d143f26e8f61feadd3d780dae251047e

**Comment by sorrison** [comment]
Fixed in 370a1f6a943c5b870ea027a0d1ea5728b5b236cf

**Pull request closed without merging**